### DISCUSSION TITLE
wav/mp3 files eg. guitar tracks playback within Hydrogen software package

### DISCUSSION THREAD

**Discussion by edwardinaday** [hydrogen-music/hydrogen#1972] [Hydrogen on Linux]
I want to import some guitar work into hydrogen although that might not be possible
right now I am clicking play on Audacity and then Hydrogen to play audio and then stopping both to start over when I want to add different drumkit sounds
is it possible to control both play pause and stop functions for both pieces in Hydrogen software?
is there a plugin or likewise to just have the play stop button in Hydrogen work for a sample like guitar tracks?
not sure how to trigger synchronous playback of both the Song in Hydrogen and the track in Audacity.

**Comment by theGreatWhiteShark** [comment]
Hey @edwardinaday ,

As you're on Linux I would highly recommend you use the `JACK` audio driver. It allows for all your audio programs to both keep their transport in sync and exchange audio seamlessly. Just install it from your distribution's repo along with the graphical interface `QJackCtl`. There are loads of guides about configuration out there. If you do not know where to start, skim through https://linuxmusicians.com/ 

An alternative would be to export the track from Audacity and load it into Hydrogen's [playback track](http://hydrogen-music.org/documentation/manual/manual_en.html#chpt.song_editor.playbackTrack).

**Comment by edwardinaday** [comment]
the EDIT button does load the track, Thank you for your informative reply
I do notice mp3 is not supported yet .wav works fine as well as .ogg
Thank you again.

**Comment by edwardinaday** [comment]
would a suggested modification to the text for the button that has EDIT on it be posted here?
something like INPUT or something... IINPUTT , TRACK-IN , ...

**Reply by theGreatWhiteShark** [reply]
Changing labels is always a quite tricky task. I agree that "edit" might not be the best choice. But altering the string would invalidate all of its translations. As translators only contribute rarely to Hydrogen we try to change as little of those strings as possible. So, even if "edit" is not ideal, I would tend to keep it.
